courses/olomouc.yml

```diff
@@ -0,0 +1,318 @@
+title: Začátečnický kurz Pythonu
+subtitle: Olomouc - podzim 2024
+time: každe útery od 17:30 do 19:30
+default_time:
+  start: '17:30'
+  end: '19:30'
+timezone: Europe/Prague
+place: Edhouse, s. r. o., Masarykova třída 936/50, 779 00, Olomouc
+description: Naučte se Python vážně od začátku. Žádné předchozí znalosti nejsou potřeba.
+long_description: |
+    Vítáme tě v Začátečnickém kurzu Pythonu od PyLadies Olomouc.
+    
+    Níže najdeš materiály ke kurzu, který začíná 8. 10. 2024. 
+
+    Jednotlivé lekce jsou určeny naprostým začátečnicím. Instrukce jsou uvedeny pro operační systémy Linux, Windows i macOS.
+    
+    Stránky samotných PyLadies najdeš na http://pyladies.cz.
+
+    Domácí úkoly k jednotlivým lekcím najdeš na https://projekty.pyladies.cz/.
+
+    Zpětnou vazbu na konci každé lekce můžeš zadat na https://zpetnovazebnik.pyladies.cz/. 
+vars:
+  pyladies: true
+  coach-present: true
+sessions:
+- slug: preparation
+  serial: 0
```

Comment:
Tohle mi přijde trochu děsivé. Když se ten kurz otevře, tak se zobrazí tohle:
![image](https://github.com/user-attachments/assets/ee123837-cac1-4064-a7de-172556ba475a)

Kdybych o tom nic nevěděl a viděl, že tohle si musím sám doma nastudovat, tak nevím, jestli bych se na to nevykašlal :laughing:

V Praze mají před prvním kurzem jenom install a install-editor, to by možná stačilo.

A co každopádně musíme upravit je přidat cmdline, install, venv-setup i install-editor na první lekci. Na první lekci se (alespoň z naších zkušeností) víceméně neprogramuje, je fakt potřeba všechno nastavit :laughing:

Reply:
> Kdybych o tom nic nevěděl a viděl, že tohle si musím sám doma nastudovat, tak nevím, jestli bych se na to nevykašlal 😆

Tak to može byť prvé sito. ;) Proti tomuto nič nemám. Určite by bolo dobre aby si zvykli hneď zo začiatku toho bude hodne. Hlavne je doležite aby sa vedeli kam sa maju obratiť ak si chcu niečo naštudovať. Ja by som to nemenil.

Reply:
Jasné, tohle nemá správnou odpověď :) Jsi hlavní kouč, takže je to na tobě.

Reply:
To nastavenie prostredia hodím na prvú hodin. Lebo je to take viac étericke.